src/main/java/core/levels/LevelManager.java

```diff
@@ -20,18 +22,66 @@
     private final OrthogonalTiledMapRenderer mapRenderer;
     private final WorldManager worldManager;
     private final SpriteBatch batch;
+    private float spawnTime;
 
     public LevelManager(LevelState level) {
         this.level = level;
         map = new TmxMapLoader().load(level.getMapPath());
         this.mapRenderer = new OrthogonalTiledMapRenderer(map, level.getUnitScale());
         this.worldManager = new WorldManager(level.world);
-
         this.batch = new SpriteBatch();
+        this.spawnTime = 15;
     }
 
+    /**
+     * Step physics simulation of World
+     * Elapse time in World
+     * Spawn enemies in World
+     *
+     * Above actions only performed if level isn't in a paused state
+     *
+     * @param dt time delta to simulate (seconds) (capped at .5 in case computer is too slow)
+     */
     public void step(float dt) {
-        worldManager.step(dt);
+        // If level isn't paused, perform step, time elapsing, spawning
+        if (!level.getLevelPaused()) {
```

Comment:
maybe just return if the level is paused, just reduces the indentation level for the main code

Reply:
don't think i understand: do you mean change to "if (level.getLevelPaused())"? in which case nothing happens when level is paused so would just end up having an empty if body?

Reply:
return statement in the if statement